src/components/UserProfile/UserSettings/index.tsx

```diff
@@ -67,50 +63,20 @@ const UserSettings: React.FC = ({ children }) => {
     },
     {
       text: intl.formatMessage(messages.menuNotifications),
-      route: '/settings/notifications',
+      route: data?.emailEnabled
+        ? `/users/${user?.id}/settings/notifications/email`
+        : `/users/${user?.id}/settings/notifications/discord`,
```

Comment:
What if neither are enabled? Should we show the discord notification page if discord notifications aren't enabled?

Reply:
The user can always enter a Discord ID, even if Discord notifications are not enabled. Some people may be using Requestrr and not have our built-in Discord notifications enabled.

Reply:
Ah yes, Requestrr! Makes sense!